app/models/specialist_document.rb

```diff
@@ -86,7 +86,11 @@ def update(params)
   def published?
     editions.any?(&:published?)
   end
-  alias_method :has_ever_been_published?, :published?
+
+  def has_ever_been_published?
+    return false if @editions.size == 1 && needs_exporting?
+    published?
```

Comment:
Ok. I don't have any concrete suggestions to offer, so I'm happy to go ahead with this.